hummingbot/strategy/fixed_grid/fixed_grid_config_map.py

```diff
@@ -41,7 +41,7 @@ def validate_price_floor_ceiling(value: str) -> Optional[str]:
 
 
 def exchange_on_validated(value: str):
-    required_exchanges.append(value)
+    required_exchanges.add(value)
```

Comment:
@petioptrv please double check this change is intended to be part of the PR. It seems unrelated.

Reply:
Discussed with @aarmoa elsewhere, but leaving the comment here for completeness: this is not related to the Eve connector, but is a quick-fix spotted during an investigation